### PR TITLE
Added documentation for the `--command` flag

### PR DESCRIPTION
**Pull request by abs007**
Reference pr: https://github.com/okteto/okteto/pull/3215. I also changed the position of the `--env` flag to match the [`up.go`](https://github.com/okteto/okteto/blob/master/cmd/up/up.go#L366) file 

Signed-off-by: Abhisman Sarkar <abhisman.sarkar@gmail.com>

### CODE REVIEW

File: src/content/reference/cli.mdx [hunk 1]
```diff
@@ -662,9 +662,10 @@ Therefore, the development deployment uses the same service account, environment
 | _--file_      | (string) | The path to the manifest file (default "okteto.yml")                                                 |
 | _--namespace_ | (string) | The namespace to use (defaults to the current okteto namespace)                                      |
 | _--context_   | (string) | The context to use (defaults to the current okteto context)                                          |
+| _--env_       |  (list)  | Set environment variable in the dev container.                                                       |
 | _--deploy_    |  (bool)  | Force execution of the commands in the `deploy` section of the okteto manifest (defaults to `false`) |
 | _--reset_     |  (bool)  | Resets the file synchronization service. Use it if the file synchronization service stops working.   |
-| _--env_       |  (list)  | Set environment variable in the dev container.                                                            |
+| _--command_   |  (list)  | Provide additional commands to override the ones specified under the dev field in the manifest       |
```
Comment (rberrelleza) [inline src/content/reference/cli.mdx:668]:
```suggestion
| _--command_   |  (list)  | Override the command specified in the `dev` section of the manifest |
```

Comment (rberrelleza) [reply src/content/reference/cli.mdx:668]: Let me know  if you agree @abs007 ?

Comment (abs007) [reply src/content/reference/cli.mdx:668]: This seems good. I'll apply the changes

**Review by pchico83 — APPROVED** [review]
Thanks @abs007, you rock!

**Comment by abs007** [comment]
Appreciate it :) @pchico83

**Pull request merged**